lib/smart_proxy_remote_execution_ssh.rb

```diff
@@ -4,6 +4,8 @@
 require 'smart_proxy_remote_execution_ssh/webrick_ext'
 require 'smart_proxy_remote_execution_ssh/plugin'
 
+Proxy::Dynflow::TaskLauncherRegistry.register('ssh', Proxy::Dynflow::TaskLauncher::Batch)
```

Comment:
Maybe doing this from the plugin's after activation hook would be better?

Reply:
Done, just heads up, I've removed two more `require '*_core'`